fetch.bs

```diff
@@ -3651,6 +3651,16 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
      <!-- Technically this only applies to HTTP/1.1 and up -->
     </ol>
 
+   <li>
+    <p>If <var>httpRequest</var>'s <a for=request>header list</a> <a for="header list">contains</a>
+    `<code>Range</code>`, then <a for="header list">append</a>
+    `<code>Accept-Encoding</code>`/`<code>identity</code>` to <var>httpRequest</var>'s
+    <a for=request>header list</a>.
+
+    <p class="note no-backref">
+    <a href="https://jakearchibald.github.io/accept-encoding-range-test/">Many servers</a>
```

Comment:
You cannot have a newline before `<a` here. We only allow whitespace at the end of paragraphs.